.github/workflows/tests_with_latest_deps.yaml

```diff
@@ -109,7 +109,7 @@ jobs:
           $ProgressPreference = "silentlyContinue"
           Invoke-WebRequest -Uri $Uri -Outfile "$env:USERPROFILE/$File"
           $hashFromFile = Get-FileHash "$env:USERPROFILE/$File" -Algorithm SHA256
-          $hashFromUrl = "e841557c27d438b96e09126a2b0656154a3a34bdb9d87d59ceaea67515713515"
```

Comment:
The windows tests were erroring with `Miniconda3-latest-Windows-x86_64.exe hashes do not match`m so I updated the hash with the 3.11 windowns installer hash at https://docs.anaconda.com/free/miniconda/miniconda-other-installer-links/